extern/sector-storage/stores/remote.go

```diff
@@ -357,6 +357,67 @@ func (r *Remote) readRemote(ctx context.Context, url string, spt abi.RegisteredS
 	return resp.Body, nil
 }
 
+func (r *Remote) CheckAllocated(ctx context.Context, s storage.SectorRef, offset, size abi.PaddedPieceSize, ft storiface.SectorFileType) (bool, error) {
+	if ft != storiface.FTUnsealed {
+		return false, xerrors.Errorf("CheckAllocated only supports unsealed files")
```

Comment:
If the method only supports unsealed files, then maybe we don't need to pass the ft parameter at all (and we can change the method name to something like CheckUnsealedAllocated)

Reply:
This is done.